### ISSUE TITLE
References in contentprovider-sample sometimes don't show

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#74482]
Ubuntu 19.04

https://github.com/microsoft/vscode/issues/74311

With https://github.com/microsoft/vscode-extension-samples/tree/master/contentprovider-sample sometimes no references would show. Not sure if this is an extension problem or not?

![image](https://user-images.githubusercontent.com/2193314/58500933-1d39f500-8138-11e9-84f9-a5fb80a26d3f.png)

No errors in console, nothing interesting in ext host output. I also saw this in the vscode codebase, didn't happen 100% of the time.

**Comment by sana-ajani** [comment]
+1 seeing this on windows too

**Comment by jrieken** [comment]
Yeah, that's an issue with the extension... Didn't know it's that popular

**Comment by Tyriar** [comment]
@jrieken should it be moved to the extension samples repo or closed then?
